playbooks/edx_amc.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+- name: Configure instance(s)
+  hosts: edx
+  sudo: True
+  gather_facts: True
+  vars:
+    migrate_db: "yes"
+    openid_workaround: True
+    EDXAPP_LMS_NGINX_PORT: '80'
+    edx_platform_repo: 'https://github.com/appsembler/edx-platform.git'
+    edx_platform_version: 'appsembler/amc/develop'
+    # Set to false if deployed behind another proxy/load balancer.
+    NGINX_SET_X_FORWARDED_HEADERS: True
+    # These should stay false for the public AMI
+    COMMON_ENABLE_DATADOG: False
+    COMMON_ENABLE_SPLUNKFORWARDER: False
+  vars_files:
+    - vault/production_secret_vars.yml
+  tasks:
+    - name: install memcached
+      apt: pkg=memcached state=present
+  roles:
```

Comment:
It runs on the same machine as the rest of edX (LMS/CMS), I'm just saying that we don't really need the ecommerce role for the AMC Alpha release.